react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -100,6 +118,10 @@ class InfoDialogButton extends Component {
         if (this.props._shouldAutoClose) {
             this._setAutoCloseTimeout();
         }
+
+        if (!this.props._dialInNumbers) {
+            this.props.dispatch(updateDialInNumbers());
+        }
```

Comment:
Previously the InfoDialog popup would fetch dial in numbers itself. When numbers are fetched the InfoDialog displayed dial in numbers, making itself taller. This was fine in the vertical toolbar case as the InfoDialog has plenty of room to grow vertically. With the new horizontal toolbar, that growing caused overlap. The fetching of numbers was moved up one level so that InlineDialog could receive new props and know to update its window location.